<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–í–æ—Å–ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏–µ —Ç–µ–∫—Å—Ç–∞ </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #0a0a0a, stop:0.3 #1a1a2e, stop:0.7 #16213e, stop:1 #0f3460);
    color: #ffffff;
}

QWidget {
    background: transparent;
    color: #ffffff;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 10pt;
}

QLabel {
    color: #00ffff;
    font-weight: bold;
    text-shadow: 0 0 10px #00ffff, 0 0 20px #00ffff;
}

QComboBox {
    background: rgba(0, 0, 0, 0.7);
    border: 2px solid #00ffff;
    border-radius: 8px;
    padding: 8px;
    color: #ffffff;
    font-weight: bold;
    selection-background-color: #00ffff;
    selection-color: #000000;
}

QComboBox:hover {
    border-color: #00ffff;
    box-shadow: 0 0 15px #00ffff;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 5px solid #00ffff;
    margin-right: 5px;
}

QTextEdit {
    background: rgba(0, 0, 0, 0.8);
    border: 2px solid #00ffff;
    border-radius: 10px;
    padding: 15px;
    color: #ffffff;
    font-size: 11pt;
    line-height: 1.5;
    selection-background-color: #00ffff;
    selection-color: #000000;
}

QTextEdit:focus {
    border-color: #00ffff;
    box-shadow: 0 0 20px #00ffff;
}

QSlider::groove:horizontal {
    border: 2px solid #00ffff;
    height: 8px;
    background: rgba(0, 0, 0, 0.7);
    border-radius: 4px;
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
        stop:0 #ffffff, stop:0.3 #00ffff, stop:1 #0080ff);
    border: 2px solid #00ffff;
    width: 20px;
    height: 20px;
    border-radius: 10px;
    margin: -6px 0;
    box-shadow: 0 0 10px #00ffff;
}

QSlider::handle:horizontal:hover {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
        stop:0 #ffffff, stop:0.3 #00ffff, stop:1 #00ffff);
    box-shadow: 0 0 15px #00ffff, 0 0 25px #00ffff;
}

QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #00ffff, stop:1 #0080ff);
    border-radius: 4px;
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #1a1a2e, stop:1 #16213e);
    border: 2px solid #00ffff;
    border-radius: 10px;
    padding: 12px 20px;
    color: #ffffff;
    font-weight: bold;
    font-size: 14pt;
    min-width: 30px;
    min-height: 20px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #00ffff, stop:1 #0080ff);
    color: #000000;
    box-shadow: 0 0 15px #00ffff, 0 0 25px #00ffff;
    transform: scale(1.05);
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #0080ff, stop:1 #00ffff);
    border-color: #ffffff;
    box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.5);
}

QPushButton:disabled {
    background: rgba(100, 100, 100, 0.3);
    border-color: #666666;
    color: #666666;
}

QMenuBar {
    background: rgba(0, 0, 0, 0.8);
    border-bottom: 2px solid #00ffff;
    color: #ffffff;
}

QMenuBar::item {
    background: transparent;
    padding: 8px 12px;
    border-radius: 4px;
    margin: 2px;
}

QMenuBar::item:selected {
    background: rgba(0, 255, 255, 0.3);
    border: 1px solid #00ffff;
    box-shadow: 0 0 10px #00ffff;
}

QMenuBar::item:pressed {
    background: rgba(0, 255, 255, 0.5);
    border: 1px solid #00ffff;
}

QMenu {
    background: rgba(0, 0, 0, 0.9);
    border: 2px solid #00ffff;
    border-radius: 8px;
    padding: 5px;
    color: #ffffff;
    font-weight: bold;
}

QMenu::item {
    background: transparent;
    padding: 8px 20px;
    border-radius: 4px;
    margin: 2px;
}

QMenu::item:selected {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #00ffff, stop:1 #0080ff);
    color: #000000;
    box-shadow: 0 0 10px #00ffff;
}

QMenu::separator {
    height: 2px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 transparent, stop:0.5 #00ffff, stop:1 transparent);
    margin: 5px 10px;
}

QStatusBar {
    background: rgba(0, 0, 0, 0.8);
    border-top: 2px solid #00ffff;
    color: #00ffff;
}

QToolButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #1a1a2e, stop:1 #16213e);
    border: 2px solid #00ffff;
    border-radius: 8px;
    padding: 8px 12px;
    color: #ffffff;
    font-weight: bold;
    font-size: 10pt;
}

QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #00ffff, stop:1 #0080ff);
    color: #000000;
    box-shadow: 0 0 15px #00ffff;
}

QToolButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #0080ff, stop:1 #00ffff);
    border-color: #ffffff;
    box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.5);
}

QToolButton:disabled {
    background: rgba(100, 100, 100, 0.3);
    border-color: #666666;
    color: #666666;
}

QListView {
    background: rgba(0, 0, 0, 0.8);
    border: 2px solid #00ffff;
    border-radius: 10px;
    padding: 10px;
    color: #ffffff;
    font-size: 11pt;
    selection-background-color: #00ffff;
    selection-color: #000000;
    outline: none;
}

QListView::item {
    background: transparent;
    padding: 8px 12px;
    border-radius: 6px;
    margin: 2px 0;
}

QListView::item:hover {
    background: rgba(0, 255, 255, 0.2);
    border: 1px solid #00ffff;
}

QListView::item:selected {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #00ffff, stop:1 #0080ff);
    color: #000000;
    font-weight: bold;
    box-shadow: 0 0 10px #00ffff;
}

QListView::item:selected:active {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #0080ff, stop:1 #00ffff);
}

QSplitter::handle {
    background: #00ffff;
    width: 3px;
}

QSplitter::handle:hover {
    background: #ffffff;
    box-shadow: 0 0 10px #00ffff;
}

QStatusBar {
    background: rgba(0, 0, 0, 0.8);
    border-top: 2px solid #00ffff;
    color: #00ffff;
    font-weight: bold;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–ö–∞—Ç–µ–≥–æ—Ä–∏—è:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="catList"/>
      </item>
      <item>
       <widget class="QToolButton" name="newCat">
        <property name="toolTip">
         <string>–°–æ–∑–¥–∞—Ç—å –Ω–æ–≤—É—é –∫–∞—Ç–µ–≥–æ—Ä–∏—é (–ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏–µ)</string>
        </property>
        <property name="text">
         <string>üìù –î–æ–±–∞–≤–∏—Ç—å –∫–∞—Ç–µ–≥–æ—Ä–∏—é</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>362</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QListView" name="textsList"/>
        <widget class="QTextEdit" name="textBrowser">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>–í–≤–µ–¥–∏—Ç–µ —Ç–µ–∫—Å—Ç –∑–¥–µ—Å—å...</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>–°–ø–∏—Å–æ–∫ –≥–æ–ª–æ—Å–æ–≤:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="VoicesList">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>–°–∫–æ—Ä–æ—Å—Ç—å –≤–æ—Å–ø—Ä–æ–∏–∑–≤–µ–¥–µ–Ω–∏—è:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ValueSpeed">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PrintValueSpeed">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="BtnPrevious">
        <property name="minimumSize">
         <size>
          <width>74</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>78</height>
         </size>
        </property>
        <property name="text">
         <string>‚è™</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnStop">
        <property name="minimumSize">
         <size>
          <width>74</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>78</height>
         </size>
        </property>
        <property name="text">
         <string>‚èπÔ∏è</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnPausePlay">
        <property name="minimumSize">
         <size>
          <width>74</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>78</height>
         </size>
        </property>
        <property name="text">
         <string>‚èØÔ∏è</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnNext">
        <property name="minimumSize">
         <size>
          <width>74</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>78</height>
         </size>
        </property>
        <property name="text">
         <string>‚è©</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>38</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>–§–∞–π–ª</string>
    </property>
    <addaction name="ActExport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>–°–ø—Ä–∞–≤–∫–∞</string>
    </property>
    <addaction name="ActAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ActOpen">
   <property name="text">
    <string>üìÇ –û—Ç–∫—Ä—ã—Ç—å</string>
   </property>
   <property name="iconText">
    <string>üìÇ –û—Ç–∫—Ä—ã—Ç—å</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="ActExport">
   <property name="text">
    <string>üíæ –≠–∫—Å–ø–æ—Ä—Ç</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ActExit">
   <property name="text">
    <string>üö™ –í—ã—Ö–æ–¥</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="ActAbout">
   <property name="text">
    <string>–û –ø—Ä–æ–≥—Ä–∞–º–º–µ üí°</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
